resources/sip11/ra/src/main/java/org/mobicents/slee/resource/sip11/wrappers/DialogWrapper.java

```diff
@@ -761,6 +767,11 @@ public Dialog getWrappedDialog() {
 	public void setWrappedDialog(Dialog wrappedDialog) {
 		this.wrappedDialog = wrappedDialog;
 		if (wrappedDialog != null) {
+			try {
+				wrappedDialog.terminateOnBye(this.terminateOnByeCached);
+			} catch (SipException e) {
+				// FIXME: what we can do when SipException occurs?
```

Comment:
ok, I think that I can use Tracer in DialogWrapper as we use Tracer in ClientDialogWrapper.